utils/api/configurations.js

```diff
@@ -53,84 +60,43 @@ export const configureRequests = ({ data, path }) => {
   })
 }
 
-const normalizeHtmlDescription = (text) => {
-  const regex = / style="white-space: pre-line;"/g
-  return text.replace(regex, '')
-}
-
-const normalizeDescription = (text) => {
-  // removes html elements, new lines, html comments and sections with 3 spaces from the string
-  const regex = /(<([^>]+)>)|\n|-+>|   /g
-  const description = text.replace(regex, '')
-  const length = 350
-
-  return description.length > length
-    ? `${description.substring(0, length - 3)}...`
-    : description
-}
-
-export const normalizeDate = (str) => {
-  if (str) {
-    const date = new Date(str)
-    return `${date.toDateString().substring(3)} at ${date.toLocaleTimeString()}`
-  }
-
-  return 'No deadline set'
-}
-
-export const timeSince = function(date) {
-  if (typeof date !== 'object') {
-    date = new Date(date)
-  }
-
-  let seconds = Math.floor((new Date() - date) / 1000)
-  let intervalType
-
-  let interval = Math.floor(seconds / 31536000)
-  if (interval >= 1) {
-    intervalType = 'year'
-  } else {
-    interval = Math.floor(seconds / 2592000)
-    if (interval >= 1) {
-      intervalType = 'month'
-    } else {
-      interval = Math.floor(seconds / 86400)
-      if (interval >= 1) {
-        intervalType = 'day'
-      } else {
-        interval = Math.floor(seconds / 3600)
-        if (interval >= 1) {
-          intervalType = 'hour'
-        } else {
-          interval = Math.floor(seconds / 60)
-          if (interval >= 1) {
-            intervalType = 'minute'
-          } else {
-            interval = seconds
-            intervalType = 'second'
-          }
-        }
+// takes an array of errors for each page or component
+export const configureErrors = (errors) => {
+  const env = process.env.NODE_ENV
+  let errorText = []
+  let errorTitle = ''
+  if (env == 'development') {
+    errors.map(error => {
+      errorText.push(JSON.stringify(error))
+      errorTitle = errors.length > 1 ? 'There were multiple errors.' : error.name
```

Comment:
if `errors = [{ title: 'some', message: 'data' }, {}]`, then we only have 1 error and not multiple. this code doesn't account for that scenario as is.